### ISSUE TITLE
Main is broken

### ISSUE THREAD

**Issue by Korijn** [pygfx/pygfx#487]
The last pylinalg release broke pygfx main :)

```
❯ .venv/Scripts/python .\examples\feature_demo\look_at.py
QWindowsWindow::setDarkBorderToWindow: Unable to set dark window border.
Traceback (most recent call last):
  File "C:\Users\kvang\dev\pygfx\examples\feature_demo\look_at.py", line 55, in <module>
    disp.show(group)
  File "c:\users\kvang\dev\pygfx\pygfx\utils\show.py", line 192, in show
    self.camera.show_object(object, up=up, scale=1.4)
  File "c:\users\kvang\dev\pygfx\pygfx\cameras\_perspective.py", line 379, in show_object
    view_dir = la.quaternion_rotate((0, 0, -1), rotation)
AttributeError: module 'pylinalg' has no attribute 'quaternion_rotate'
```

See https://github.com/pygfx/pylinalg/pull/53

Replacing every call to `la.quaternion_rotate` with `la.vector_apply_quaternion` fixes the problem.

**Comment by FirefoxMetzger** [comment]
Yes, This goes back to the `vector_apply_quaternion` PR where we decided to nuke `quaternion_rotate` in favor of `vector_apply_quaternion` 😅

**Comment by Korijn** [comment]
Fixed in #488
